state/mysql/mysql.go

```diff
@@ -523,6 +535,34 @@ func (m *MySQL) Multi(request *state.TransactionalStateRequest) error {
 	return nil
 }
 
+// Returns the set requests.
+func (m *MySQL) getSets(req state.TransactionalStateOperation) (state.SetRequest, error) {
```

Comment:
Rename as `getSetRequest`

Reply:
This is the same convention we have been following for other state stores. Should we keep it for the sake of uniformity, or do you think it's better to update? Same is valid for the `getDeletes` comment.

For example https://github.com/dapr/components-contrib/blob/99df8741ea84ffcb973441efa858a26e8a46dda2/state/sqlserver/sqlserver.go#L357

Reply:
OK.